### ISSUE TITLE
Opening PDF fails with "Stream cannot be read. Please send us the PDF file so that we can fix this"

### ISSUE THREAD

**Issue by mikethea1** [empira/PDFsharp#207]
# Reporting an Issue Here
## Expected Behavior

PDF opens without error

## Actual Behavior

Exception:
```
System.InvalidOperationException: Stream cannot be read. Please send us the PDF file so that we can fix this (issues (at) pdfsharp.net).
   at PdfSharp.Pdf.IO.Lexer.ScanStream(Int64 position, Int32 length)
   at PdfSharp.Pdf.IO.Parser.ReadDictionaryStream(PdfDictionary dict, SuppressExceptions suppressObjectOrderExceptions)
   at PdfSharp.Pdf.IO.Parser.ReadObjectInternal(PdfObject pdfObject, PdfObjectID objectID, Boolean includeReferences, Boolean fromObjectStream, SuppressExceptions suppressObjectOrderExceptions)
   at PdfSharp.Pdf.IO.Parser.ReadIndirectObject(PdfReference pdfReference, SuppressExceptions suppressObjectOrderExceptions, Boolean withoutDecrypting)
   at PdfSharp.Pdf.IO.Parser.ReadAllIndirectObjects()
   at PdfSharp.Pdf.IO.PdfReader.OpenFromStream(Stream stream, String password, PdfDocumentOpenMode openMode, PdfPasswordProvider passwordProvider, PdfReaderOptions options)
   at PdfSharp.Pdf.IO.PdfReader.OpenFromFile(String path, String password, PdfDocumentOpenMode openMode, PdfPasswordProvider passwordProvider)
   at PdfSharp.Pdf.IO.PdfReader.Open(String path, String password, PdfDocumentOpenMode openMode, PdfPasswordProvider passwordProvider, PdfReaderOptions options)
   at PdfSharp.Pdf.IO.PdfReader.Open(String path, PdfDocumentOpenMode openMode, PdfReaderOptions options)
```

## Steps to Reproduce the Behavior
[Issue.zip](https://github.com/user-attachments/files/17662782/Issue.zip)

**Comment by ThomasHoevel** [comment]
The file is corrupted.

Next preview returns a better error message:

---------------------------
PDFsharp Document Explorer
---------------------------
Stream cannot be read.
Position: 9860663, expected length: 1654, bytes read: 1259, PDF length: 9861922.
The PDF file is corrupted, stream goes beyond actually file length.

---------------------------
OK   
---------------------------


Object 44 is empty (0 bytes), but reported size is 1654. Maybe buggy scanner software.
```
44 0 obj
<</Length 1654
/Type
/Metadata
/Subtype 
/XML>>
stream
endstream
endobj
45 0 obj

```

**Comment by mikethea1** [comment]
@ThomasHoevel the file opens just fine for me in Chrome and PdfPig can also parse it. Are you seeing the same thing?

**Comment by ThomasHoevel** [comment]
PDFsharp attempts to read all objects and stumbles at the corrupted object 44.
Other tools may ignore this object.

**Comment by mikethea1** [comment]
@ThomasHoevel can PDFSharp be configured to ignore?

**Comment by ThomasHoevel** [comment]
> can PDFSharp be configured to ignore?

Not yet. But if PDFsharp can ignore this nonsense with little effort, we'll change PDFsharp to ignore that garbage.

**Comment by ThomasHoevel** [comment]
Issue should be resolved with PDFsharp 6.2.0 Preview 2.

**Comment by mikethea1** [comment]
Thanks @ThomasHoevel !
